repo/content/content_manager_indexes.go

```diff
@@ -31,16 +31,16 @@ func (bm *Manager) CompactIndexes(ctx context.Context, opt CompactOptions) error
 		return errors.Wrap(err, "error loading indexes")
 	}
 
-	contentsToCompact := bm.getContentsToCompact(ctx, indexBlobs, opt)
+	blobsToCompact := bm.getBlobsToCompact(ctx, indexBlobs, opt)
 
-	if err := bm.compactAndDeleteIndexBlobs(ctx, contentsToCompact, opt); err != nil {
+	if err := bm.compactAndDeleteIndexBlobs(ctx, blobsToCompact, opt); err != nil {
```

Comment:
nit: since this is no longer deleting index blobs, perhaps rename the function?
```suggestion
	if err := bm.compactIndexBlobs(ctx, blobsToCompact, opt); err != nil {
```

Reply:
renamed